src/pages/home/sidebar/OptionRowTitle/index.js

```diff
@@ -0,0 +1,120 @@
+            return;
+        }
+        const {width: cWidth, left: cLeft} = containerLayout;
+
+        // we have to return the value as Number so we can't use `measureWindow` which takes a callback
+        const {width: tWidth, left: tLeft} = this.cRefs[index].getBoundingClientRect();
+        const toolX = (tWidth / 2) + tLeft;
+        const cRight = cWidth + cLeft;
+        const tRight = tWidth + tLeft;
+        const newToolX = tLeft + ((cRight - tLeft) / 2);
+
+        // when text right end is beyond the Container Right end
+        return tRight > cRight ? -(toolX - newToolX) : 0;
+    }
+
+
+    render() {
+        const {
+            option, style, tooltipEnabled, numberOfLines, tooltipContainerStyle,
+        } = this.props;
+        const {isEllipsisActive} = this.state;
+
+        if (tooltipEnabled) {
+            return (
+
+                // Tokenization of string only support 1 numberofLines on Web
+                <Text
+                    style={[style, styles.optionDisplayNameTooltipWrapper]}
+                    onLayout={this.setContainerLayout}
+                    numberOfLines={1}
+                    ref={this.setContainerRef}
+                >
+                    {_.map(option.participantsList, (participant, index) => {
+                        const setChildRef = this.setOptionChildRef(index);
+                        return (
+                            <Fragment key={index}>
+                                <Tooltip
+                                    key={index}
+                                    text={participant.login}
+                                    containerStyle={tooltipContainerStyle}
+                                    shiftHorizontal={() => this.getTooltipShiftX(index)}
+                                >
+                                    <Text ref={setChildRef}>
+                                        {participant.displayName}
+                                    </Text>
+                                </Tooltip>
+                                {index < option.participantsList.length - 1 ? <Text>,&nbsp;</Text> : null}
+                            </Fragment>
+                        );
+                    })}
+                    {
+                        option.participantsList.length > 1 && isEllipsisActive
+                            ? (
+                                <View style={styles.optionDisplayNameTooltipEllipsis}>
+                                    <Tooltip text={option.tooltipText}>
+                                        {/* there is some Gap for real ellipsis so we are adding 4 `.` to cover */}
+                                        <Text>....</Text>
+                                    </Tooltip>
+                                </View>
+                            ) : null
+                    }
+                </Text>
+            );
+        }
+
+        return <Text style={style} numberOfLines={numberOfLines}>{option.text}</Text>;
```

Comment:
Move this case to be the first case inside an `if (!tooltipEnabled)` so the default case is more clear.